pgt/src/PGT/Output/Test/Result/examples/empty-json-records.expected

```diff
@@ -0,0 +1,4 @@
+-[ RECORD 1 ]
+failure |
+jsonb   | [+
+        | ]
```

Comment:
__question:__ Could the test be added prior to the commit so that we can see the difference the code change makes?

Reply:
@dkubb Good catch. I will add it prior to the code changes.